ext/standard/tests/file/file_put_contents_5gb.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+Test file_put_contents() function with 5GB string
+--SKIPIF--
+<?php
+if (getenv('SKIP_SLOW_TESTS')) {
+    die('skip slow test');
+}
+function get_system_memory(): int|float|false
+{
+    if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+        // Windows-based memory check
+        @exec('wmic OS get FreePhysicalMemory', $output);
+        if (isset($output[1])) {
+            return (int)trim($output[1]);
+        }
+    } else {
+        // Unix/Linux-based memory check
+        $memInfo = @file_get_contents("/proc/meminfo");
+        if ($memInfo) {
+            preg_match('/MemFree:\s+(\d+) kB/', $memInfo, $matches);
+            return $matches[1] * 1024; // Convert to bytes
```

Comment:
I guess this is now wrong on Linux, right?

Reply:
hmm... yes it is. fsck it, ill fix it

Reply:
thanks fixed c8420b18696fc48d6f17e7fa7b26978ef0c6b7ea